cms/templates/widgets/footer.html

```diff
@@ -31,11 +30,13 @@
             % if waffle().is_enabled(ENABLE_ACCESSIBILITY_POLICY_PAGE):
               <li class="nav-item nav-peripheral-aar">
                 <a data-rel="edx.org" href="${reverse('accessibility')}">${_("Accessibility Accommodation Request")}</a>
-              </li>
+              </li> 
             %endif
+            <li class="nav-item nav-peripheral-aar">
```

Comment:
This re-uses the CSS class from the accessibility accomodation request.  I don't understand what these classes are for, but it doesn't seem right to re-use it.

Reply:
Ah, yes. That's not right. I don't see these nav-peripheral-* classes being used, so I think I will remove the class.